### ISSUE TITLE
LoRA Recomendations / Guide

### ISSUE THREAD

**Issue by MrKuenning** [bmaltais/kohya_ss#62]
I am very excited about the concept of LoRA because I can quickly train on lots of family and friends and not take up several hundred more GB on my system.  Also, you can apply the trained LoRA to other models on the fly.

I have trained about 15 or so models so far and have "decent" results.  Faces look accurate if there is no other words in the prompt and sometimes start to look less and less like the person the longer the prompt gets.

Also, they seem to zero in on one outfit from one image in the bunch and use that over and over.

I would love some tricks or recommended settings for LoRA people training to get the best results.

How many images?
How many repeats?
Learning rate etc...

I have been using SD 1.5 model,  about 10 cropped faces at 512x512 with 160 repeats and everything else left default.

**Comment by bmaltais** [comment]
I have been training some LoRA with 100 and another with 800 images. I have had some pretty good results.

For the 100 image dataset I usually use 50 repeats with TE LR of 5e-5 and UNet LR of 1e-3. I also enable flip and color augmentation. I also use WD14 captions with some tweaking and enable shuffle captions.

**Comment by LazyCat420** [comment]
If I want to get more epochs/checkpoints to save would it be better to increase batch size or increase epochs?
I'm guessing if I have the vram I could increase the batch size but then I have to increase the training rate by the same multiple. 
If I increase epochs should I divide the repeats by the number of epochs?
Because with this version there's no steps so I'm kind of confused.
